tests/test_visitors/test_ast/test_loops/test_loops/test_infinite_while_loops.py

```diff
@@ -50,6 +50,36 @@ def wrapper():
     {1}
 """
 
+correct_loop_with_try1 = """
+while True:
+   try:
+       some_function_that_raises()
+   except Exception:
+       print('Shut down')
```

Comment:
Please, substitute all lines to `...`, it will indicate that real names and values do not matter.